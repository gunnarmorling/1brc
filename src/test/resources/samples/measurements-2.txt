Bosaso;19.2
Petropavlovsk-Kamchatsky;9.5
