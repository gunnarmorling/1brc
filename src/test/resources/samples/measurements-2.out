{Bosaso=19.2/19.2/19.2, Petropavlovsk-Kamchatsky=9.5/9.5/9.5}
